laurent22/joplin
# Desktop: Fixes #9960: Creating a profile changes the language of Joplin

### Summary
This fixes the issue where creating a new profile changes the languages of all profiles including the default profile to the system default language.

Fixes #9960.

### Testing
Follow reproduction steps in #9960:

1. Change your Joplin language other than the operating system language
2. Create a new profile
3. The language for the new profile is the same as for the original profile
4. Change back to your original profile
5. Verify that the language of the original profile has not changed

This has been tested successfully on Windows 11 and Ubuntu.
